packages/form-builder/src/blocks/fields/index.js

```diff
@@ -10,13 +10,25 @@ import DonorName from "./donorName";
 const fieldBlocks = [
```

Comment:
Would like to see this list of blocks broken out into their own files so we can just return a list of field names instead of a really long config.

```
const fieldBlocks = [
  textField,
  donorNameField,
  emailField
]
```

Reply:
Resolved by https://github.com/impress-org/givewp-next-gen/pull/39/commits/215de048f86d35af32090d7a99cc2f52abdde377